staging/src/k8s.io/client-go/discovery/discovery_client.go

```diff
@@ -391,13 +391,6 @@ func (d *DiscoveryClient) OpenAPISchema() (*openapi_v2.Document, error) {
 	data, err := d.restClient.Get().AbsPath("/openapi/v2").SetHeader("Accept", mimePb).Do().Raw()
 	if err != nil {
 		if errors.IsForbidden(err) || errors.IsNotFound(err) || errors.IsNotAcceptable(err) {
-			// single endpoint not found/registered in old server, try to fetch old endpoint
-			// TODO(roycaihw): remove this in 1.11
```

Comment:
As far as I know, kubectl/client-go can currently be used against 1.9+ servers. Removing this would mean it would only work against 1.10+ servers. This fallback is not costly to maintain, so I would tend to leave it something that requires significant maintenance requires us to drop it.

Reply:
I can live with that. What about the downloader for aggregated servers?

Reply:
updated release note, lgtm